widget.subscribe.php

```diff
@@ -413,6 +415,36 @@ function add_subscriber() {
 				}
 			}
 
+			if ( ! empty( $sailthru['google_recaptcha_site_key'] ) && ! empty( $sailthru['google_recaptcha_secret'] ) && isset ( $_POST['captcha_token'] ) && ! empty( $_POST['captcha_token'] ) ) {
+				write_log( "reCaptcha enabled, verifying" );
+				try {
+					$response = wp_remote_get( 'https://www.google.com/recaptcha/api/siteverify?secret=' . $sailthru['google_recaptcha_secret'] . '&response=' . $_POST['captcha_token'] );
```

Comment:
Can we sanitize the `$_POST` object: https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/#validation-sanitization-and-escaping

Reply:
@brownac I think you need to be using `$recaptcha_token` instead of `$_POST['captcha_token']` here.